orchidui/src/ui/Btn/Btn.js

```diff
@@ -0,0 +1,4 @@
+import Btn from './Btn.vue'
+export {
+    Btn
```

Comment:
should we call this `Button` or `Btn`?

Reply:
Yes, sorry it hasn't been updated. will change its name to Button

Reply:
I think we should have a prefix. `OrchidButton`, because:
- Button will be same as semantic html tag
- We also have a rule for naming component in Vue. Component should have multiple words https://eslint.vuejs.org/rules/multi-word-component-names.html

Reply:
Prefix will be good if its a complex component like "OrchidCustomerCell" but for basic components like buttons and inputs adding "Orchid" prefix will make it very hard for developers to keep typing the prefix everytime they want to use it

Reply:
fixed update btn to Button